app/models/mod_note.rb

```diff
@@ -1,7 +1,7 @@
 # typed: false
 
 class ModNote < ApplicationRecord
-  extend TimeAgoInWords
+  extend ActionView::Helpers::DateHelper
```

Comment:
It fields weird to extend dateHelper to ModNote. But had to do this to make the tests pass. This is because most of the methods are class methods. could they be made regular methods ?